src/src/components/update-app-project/update-app-project.jsx

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useState } from 'react';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import notify from '../../services/notify';
+import enums from '../../enums/enums';
+import appProjectService from '../../services/appProjectService';
+
+
+const UpdateAppProject = () => {
+  const [appProjectType, setAppProjectType] = useState("default");
+  const [title, setTitle] = useState("");
+  const [breadcrumbTitle, setBreadcrumbTitle] = useState('');
+  const [appProjectViewId, setappProjectViewId] = useState("");
+  const [internalPort, setInternalPort] = useState("");
+  const [state, setState] = useState(true);
+  const { id } = useParams();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    appProjectService
+      .getById(id)
+      .then((resp) => {
+        // eslint-disable-next-line no-debugger
+        debugger
```

Comment:
debugger and commented line could be removed

Reply:
Done.